### ISSUE TITLE
memacs_firefox output issues

### ISSUE THREAD

**Issue by japhir** [novoid/Memacs#103]
in https://github.com/novoid/Memacs/issues/87 this issue is mentioned for chrome, and in https://github.com/novoid/Memacs/pull/98 @alex-a-soto fixed the chrome output by including a title url in the note's name.

It would be great if the firefox-backend also did this. 

Furthermore it appears that the firefox script is adding a single space before all the lines after the `## -- coding: utf-8 mode: org --`  line. I am running the AUR package and wrote the below script to run the package (it gives an error when firefox is running) and fix the first line issue.

```
#!/usr/bin/sh
pgrep -x firefox >/dev/null && echo "firefox is running" ||
        memacs_firefox -f /home/japhir/.mozilla/firefox/5sm2gv3t.default/places.sqlite | sed 's/^ \*/*/' > /home/japhir/org/memacs/mozhist.org_archive
```

**Comment by japhir** [comment]
it does look like the line https://github.com/novoid/Memacs/blob/d047c5f87ca106e35248f50cca65ee23dc231e86/memacs/firefox.py#L58 is identical though…

**Comment by novoid** [comment]
There are multiple people who contributed to the Firefox module (and who probably are using it in contrast to me):
https://github.com/novoid/Memacs/commits/master/memacs/firefox.py

Does somebody volunteer for this issue?
@kidd @nguyenhaiac

**Comment by kidd** [comment]
Unfortunately I'm not using memacs anymore and I'm not sure when I'll have the time to test this up :(

**Comment by japhir** [comment]
I also decided that waiting for several seconds for my agenda to pop up is too annoying as yet, so this feature doesn't have priority for me.

**Comment by novoid** [comment]
> I also decided that waiting for several seconds for my agenda to pop up is too annoying as yet, so this feature doesn't have priority for me.

Well, Memacs does not affect your normal agenda performance at all when you generate Memacs data to archive files as described on https://github.com/novoid/Memacs/blob/master/docs/FAQs_and_Best_Practices.org#performance-and-scalability

**Comment by japhir** [comment]
Yes, I know, but for what I would actually use it: to look at what I was doing on a particular day/time, it does slow down the archive view massively. I hadn't realized before looking into this endeavour that your loading times were ~20s ;-)

**Comment by japhir** [comment]
I guess I had the naive view that "they're just a bunch of text files, there's no reason for this to be slow" without realising that the org files are quite complex to parse. I only got a real appreciation for this during the last Emacsconf :).
